app/solvers/exact_solver.py

```diff
@@ -105,63 +59,106 @@ def duplicate(items):
     return unique
 
 
-def create_all_possible_paths(graph,time):
-    lista = []
-    for x in graph.keys():
-        all_paths=[]
-        newpaths=[]
-        lista = lista + duplicate(find_all_possible_paths(graph,x,time))
-        print(x)
-    return lista
-
-
-def create_cities_dictionary(graph):
+def create_cities_dictionary(graph, cities_temp):
+    cities_dict_temp ={}
```

Comment:
czy to nie jest duplikacja funkcji z `random_solver`?

Reply:
Wygląda na to, że tak. Zresztą ten kawałek pisałem z Pszemkiem.

Reply:
To użyj tamtej 😉

Reply:
Rozumiem, że w czymś przeszkadza zdefiniowanie tego jako oddzielna funkcja?

Reply:
DRY, https://en.wikipedia.org/wiki/Don%27t_repeat_yourself